gripper_action_controller/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+cmake_minimum_required(VERSION 3.5)
+project(gripper_action_controller)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+#if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
```

Comment:
we should have this in all controllers, is there a reason you had to disable it?

Reply:
Sorry I forgot to uncomment it, I had a very long compile error regarding some templates and it was interfering with the warnings